Specs/Core/OrientedBoundingBoxSpec.js

```diff
@@ -0,0 +1,417 @@
+/*global defineSuite*/
+defineSuite([
```

Comment:
It's at 96% now, but one is the function we're removing and the others will be easy to cover.